conf/server.yaml.template

```
@@ -8,12 +8,10 @@ SERVER:
     # RELEASE: populate with satellite version
     # The snap version currently testing (if applicable)
     # SNAP:
-    # The RHEL Base OS Version(x.y) where the Satellite is installed
-    # RHEL_VERSION:
     # The source of Satellite packages. Can be one of:
     # internal, ga, beta
     SOURCE: "internal"
-    RHEL_RELEASE: '7'
+    RHEL_VERSION: '7'
```

Also, add a comment mentioning it is rhel_version of satellite, otherwise, it can create confusion
```suggestion
    # The RHEL Base OS Version(x.y) where the Satellite is installed
    RHEL_VERSION: '7'
```